msal4j-brokers/src/main/java/com/microsoft/aad/msal4jbrokers/MsalRuntimeBroker.java

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4jbrokers;
+
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import com.microsoft.aad.msal4j.IBroker;
+import com.microsoft.aad.msal4j.InteractiveRequestParameters;
+import com.microsoft.aad.msal4j.PublicClientApplication;
+import com.microsoft.aad.msal4j.SilentParameters;
+import com.microsoft.aad.msal4j.UserNamePasswordParameters;
+import com.microsoft.aad.msal4j.MsalException;
+import com.microsoft.aad.msal4j.AuthenticationErrorCode;
+import com.microsoft.aad.msal4j.IAccount;
+import com.microsoft.azure.javamsalruntime.Account;
+import com.microsoft.azure.javamsalruntime.AuthParameters;
+import com.microsoft.azure.javamsalruntime.AuthResult;
+import com.microsoft.azure.javamsalruntime.MsalInteropException;
+import com.microsoft.azure.javamsalruntime.MsalRuntimeInterop;
+import com.microsoft.azure.javamsalruntime.ReadAccountResult;
+
+import java.net.MalformedURLException;
+import java.util.Collections;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class MsalRuntimeBroker implements IBroker {
+    private static MsalRuntimeInterop interop;
+
+    static {
+        try {
+            //MsalRuntimeInterop performs various initialization steps in a similar static block,
+            // so when an MsalRuntimeBroker is created this will cause the interop layer to initialize
+            interop = new MsalRuntimeInterop();
+        } catch (MsalInteropException e) {
+            //TODO: Meaningful error message explaining why MSALRuntime failed to initialize (unsupported platform, error from MSALRuntime, etc.)
```

Comment:
Lets complete this TODO as it is very important for the app developer to understand the next steps.

Reply:
In the latest commit this throws a new exception explaining that the interop could not be initialized, and it includes the message from the MsalInteropException thrown here.